### PR TITLE
fix(database_error): stop using `exception` and use log level

### PR DESCRIPTION
**Pull request by fruch**
Since we keep on hitting on case exception is in the log text
but it's o.k., we should start looking at the log level insted to
identify error happend.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@ilya-rarov regarding your question, we might need to look for exception word, it just add us more work.

**Comment by fruch** [comment]
Taking this change for a test on master version:
https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity/job/longevity-10gb-3h/36/

**Comment by fruch** [comment]
> Taking this change for a test on master version: https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity/job/longevity-10gb-3h/36/

found issue with rpc error being raised cause of this change, after fixing it, running it again:
https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity/job/longevity-10gb-3h/37/

**Comment by fruch** [comment]
the adding of info level to journalctl tailing, work o.k. here:
https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-centos8-test/7/

**Comment by bentsi** [comment]
@fruch let's discuss before merging, we can't filter all the exceptions even if they are logged with info log level

**Comment by roydahan** [comment]
I'm ok with filtering exceptions that are logged as INFO.
The current situation just make too many warnings or errors that no one really looking at.

@fruch please fix conflicts.

**Comment by fruch** [comment]
@roydahan conflict fixed, and I've run the 3h job to verify it's working as expected

**Review by roydahan — APPROVED** [review]

**Comment by roydahan** [comment]
@bentsi / @enaydanov can one also give a quick review here?

**Comment by fruch** [comment]
@bentsi / @enaydanov ping

File: sdcm/sct_events/database.py [hunk 1]
```diff
@@ -136,8 +133,12 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                                    regex="journal: Suppressed")
 DatabaseLogEvent.add_subevent_type("stream_exception", severity=Severity.ERROR,
                                    regex="stream_exception")
-
-
+DatabaseLogEvent.add_subevent_type("RPC_CONNECTION", severity=Severity.WARNING,
```
Comment (soyacz) [inline sdcm/sct_events/database.py:136]: won't it be published every time we restart some node multiple times flooding WARNING event log?
Is there an easy way of filtering these when SCT triggers restart of some node/server?

**Review by soyacz — APPROVED** [review]
Besides one question in comment to think of and assuming it works, LGTM

File: sdcm/sct_events/database.py (hunk 1 above)
Comment (fruch) [reply sdcm/sct_events/database.py:136]: there is a way, but I don't think all the places we restart/reboot a node, are at the same place.

**Review by vponomaryov — APPROVED** [review]
LGTM

**Pull request merged**